clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -183,4 +184,64 @@ public ApiVersionsResults apiVersions(Collection<Node> nodes) {
      * @return                  The ApiVersionsResults.
      */
     public abstract ApiVersionsResults apiVersions(Collection<Node> nodes, ApiVersionsOptions options);
+
+    /**
+     * Similar to #{@link AdminClient#listAcls(AclFixture, ListAclsOptions),
+     * but uses the default options.
+     *
+     * @param filter            The filter to use.
+     * @return                  The DeleteAclsResult.
+     */
+    public ListAclsResults listAcls(AclFixture filter) {
```

Comment:
Hmm, seems like we are exposing a class from an internal package (`requests`) in a public class.